server/src/main/java/org/elasticsearch/threadpool/ThreadPool.java

```diff
@@ -1086,6 +1086,13 @@ public static boolean assertCurrentThreadPool(String... permittedThreadPoolNames
         return true;
     }
 
+    public static boolean assertTestThreadPool() {
```

Comment:
Is this used and if so does it belong in prod code?

Reply:
did this method stay after all? If it did, at least javadocs would be good to add?